Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -220,36 +231,55 @@ done < netdev.log
 			# Either VM hang or VM resume needs longer time.
 			throw "VM resume did not finish, the latest state was $state"
 		}
-		#Verify the VM status after power on event
-		$vmStatus = Get-AzVM -Name $vmName -ResourceGroupName $rgName -Status
-		if ($vmStatus.Statuses[1].DisplayStatus -eq "VM running") {
-			Write-LogInfo "$($vmStatus.Statuses[1].DisplayStatus): Verified successfully VM status is running after resuming"
-		} else {
-			Write-LogErr "$($vmStatus.Statuses[1].DisplayStatus): Could not find the VM status after resuming"
-			throw "Can not identify VM status after resuming"
-		}
 
-		# Verify the kernel panic, call trace or fatal error
-		$calltrace_filter = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "dmesg | grep -iE '(call trace|fatal error)'" -ignoreLinuxExitCode:$true
+			# Verify the VM status after VM is accessible.
+			# Read VM status from the host during 10 min-timeout
+			$timeout = New-Timespan -Minutes 10
+			$sw = [diagnostics.stopwatch]::StartNew()
+			$_verified = 0
+			while ($sw.elapsed -lt $timeout){
+				Wait-Time -seconds 15
+				$vmStatus = Get-AzVM -Name $vmName -ResourceGroupName $rgName -Status
+				if ($vmStatus.Statuses[1].DisplayStatus -eq "VM running") {
+					$_verified = 1
+					break
+				} else {
+					Write-LogDbg "$($vmStatus.Statuses[1].DisplayStatus): VM status is not 'VM running' yet. Check the next status in 15 seconds."
+				}
+			}
 
-		if ($calltrace_filter -ne "") {
-			Write-LogErr "Found Call Trace or Fatal error in dmesg"
-			# The throw statement is commented out because this is linux-next, so there is high chance to get call trace from other issue. For now, only print the error.
-			# throw "Call trace in dmesg"
-		} else {
-			Write-LogInfo "Not found Call Trace and Fatal error in dmesg"
-		}
+			if ($_verified -eq 1) {
+				Write-LogInfo "Successfully verified VM status - $vmStatus.Statuses[1].DisplayStatus"
+			} else {
+				throw "Can not find VM status after 10-min checking - $vmStatus.Statuses[1].DisplayStatus"
```

Comment:
ditto, the status can be found, but not expected.

Reply:
Fixed in ae66ef26caf39800d996c15cf19eed2b44ecfc9f